deploy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# This script updates the documentation and pushes to the corresponding
+# Github repository
+# See https://gist.github.com/domenic/ec8b0fc8ab45f39403dd
+
+# Only push when updating master (i.e. new releases)
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "master" ]; then
+    echo "Skipping deploy; just doing a build."
+    exit 0
+fi
+
+SHA=`git rev-parse --verify HEAD`
+
+# Install sphinx, and build the documentation
+pip install sphinx sphinx_rtd_theme
+cd docs
+make html
+cd ..
+
+# Get ssh credential to push to the documentation, using encrypted key
+openssl aes-256-cbc -K $encrypted_12c8071d2874_key -iv $encrypted_12c8071d2874_iv -in deploy_key.enc -out deploy_key -d
```

Comment:
`$encrypted_12c8071d2874_key` and `$encrypted_12c8071d2874_iv` are the secret environment variables that Travis CI maintains.